### PR TITLE
Auth article completed, added a screenshot in resources

### PR DESCRIPTION
**Pull request by banani720**
Feel free to recommend edits, add details, remove details, language errors etc.

### CODE REVIEW

File: content/post/2021-12-29-amplify.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - Amplify
+tags:
+  - Markdown
+  - 
+thumbnail: 
+---
+
+**Over the last couple of days,** I decided to take a bit of a dive into AWS's Amplify service and see what exactly it had to offer in terms of creating and deploying a web-based application. As Amazon describes it, "AWS Amplify is a set of purpose-built tools and features that lets frontend web and mobile developers quickly and easily build full-stack applications", so I was curious to see really how quick and easy this process was. For this article, we are going to be looking at implementing authentication using the tools the Amplify CLI provides, and try to gain understanding of what is going as we make use of it.
+
+The project I decided to use this for used an Angular front-end. As for the backend services. The goal was to use as many of the services as possible provided in the Amplify ecosystem. Setting up amplify was simple enough. First, I needed to install the Amplify CLI via npm with `npm install -g @aws-amplify/cli` then run `amplify configure`, which provided a simple step-by-step guide to getting hooked up with my AWS account and creating an IAM user for Amplify to provision resources with.
```
Comment (sharifanani) [inline content/post/2021-12-29-amplify.md]: The`As for backend services.` sentence feels misplaced, doesn't really read right. Was it meant to be there? Or is it supposed to read something like "As for backend services, the plan is to use only features offered by amplify"?

Comment (sharifanani) [inline content/post/2021-12-29-amplify.md]: For installation, show the installation for a specific version. This way your article isn't broken in the future. `npm install -g @aws-amplify/cli@<version-number>`

File: content/post/2021-12-29-amplify.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+title: Amplify
+author: Basel Anani
+date: '2021-12-29'
+categories:
+  - AWS
+  - Amplify
+tags:
+  - Markdown
+  -
```
Comment (sharifanani) [inline content/post/2021-12-29-amplify.md]:
```suggestion
  - AWS
  - Amplify
  - Angular
  - WebDev
  - Serverless
```

File: content/post/2021-12-29-amplify.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
... a polyfill, we move on to the Components section of the UI guide and select the Authenticator Component [found here](https://ui.docs.amplify.aws/components/authenticator?platform=angular). The docs do a good job highlighting the module import you need to add to your `app.module.ts` file but it's what is unhighlighted and not included that are key to getting this component to work. Asides from the import of the `AmplifyAuthenticatorModule`, an import of `aws-config from ../aws-exports` and an import of `Amplify from 'aws-amplify'` are also required. You will also need to add below the import statements `Amplify.configure(awsconfig);`. This part in particular was completely ommitted from the documentation. Failing to perform these steps before serving up your application will lead to an error about Amplify not being configured properly. Another important step to note, in your tsconfig.json, under the `compilerOptions` property, it is necessary to add `"allowJS":true`, so that the aws-exports file can be read. A complete example of all the imports and statements needed to get the component functioning properly can be found in the snippets of `app.module.ts` and `tsconfig.json` below:
```
Comment (sharifanani) [inline content/post/2021-12-29-amplify.md]: >The docs do a good job highlighting the module import you need to add to your `app.module.ts` file but it's what is unhighlighted and not included that are key to getting this component to work. Asides from the import of the `AmplifyAuthenticatorModule`, an import of `aws-config from ../aws-exports` and an import of `Amplify from 'aws-amplify'` are also required. You will also need to add below the import statements `Amplify.configure(awsconfig);`. This part in particular was completely ommitted from the documentation. Failing to perform these steps before serving up your application will lead to an error about Amplify not being configured properly.

I would immediately mention why they're required. Maybe phrase this more like:

>The docs do a good job highlighting the high-level imports you need, but leave little room for error. after _mention step from guide where you got stuck/stopped_, I ran into `actually copy and paste error here, people google their error messages all the time`. To get the component working, I needed to 
> ```
> import aws-config from '../aws-exports'
> import Amplify from 'aws-amplify'
> ```
>in `filename_here`

Comment (sharifanani) [inline content/post/2021-12-29-amplify.md]: >`"allowJS":true`

Is this truly necessary? would a `// @ts-ignore` work for that import instead? Allowing JS throughout a project is usually a last resort I would think

File: content/post/2021-12-29-amplify.md (hunk 1 above)
Comment (banani720) [reply content/post/2021-12-29-amplify.md]: Fixed

Comment (banani720) [reply content/post/2021-12-29-amplify.md]: Added version number

File: content/post/2021-12-29-amplify.md (hunk 3 above)
Comment (banani720) [reply content/post/2021-12-29-amplify.md]: Fixed

Comment (banani720) [reply content/post/2021-12-29-amplify.md]: Added justification for imports and their importance

**Pull request merged**